src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,56 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("Set의 size() 메소드로 Set 의 크기를 확인한다.")
+    @Test
+    void setSizeTest() {
+        int size = numbers.size();
+        assertThat(size).isEqualTo(3);
```

Comment:
```suggestion
        assertThat(size).hasSize(3);
```

assertj 의 api 에는 이렇게도 비교할수 있어요!